templates/chart-maps.html

```diff
@@ -1,102 +1,87 @@
+	['DeAztec',      {{.UserStatsForGame.Stats.TotalWinsMapDeAztec}},      {{.UserStatsForGame.Stats.TotalRoundsMapDeAztec}}      - {{.UserStatsForGame.Stats.TotalWinsMapDeAztec}}],
+	['DeBank',       {{.UserStatsForGame.Stats.TotalWinsMapDeBank}},       {{.UserStatsForGame.Stats.TotalRoundsMapDeBank}}       - {{.UserStatsForGame.Stats.TotalWinsMapDeBank}}],
+	['DeCbble',      {{.UserStatsForGame.Stats.TotalWinsMapDeCbble}},      {{.UserStatsForGame.Stats.TotalRoundsMapDeCbble}}      - {{.UserStatsForGame.Stats.TotalWinsMapDeCbble}}],
+	['DeDust',       {{.UserStatsForGame.Stats.TotalWinsMapDeDust}},       {{.UserStatsForGame.Stats.TotalRoundsMapDeDust}}       - {{.UserStatsForGame.Stats.TotalWinsMapDeDust}}],
+	['DeDust2',      {{.UserStatsForGame.Stats.TotalWinsMapDeDust2}},      {{.UserStatsForGame.Stats.TotalRoundsMapDeDust2}}      - {{.UserStatsForGame.Stats.TotalWinsMapDeDust2}}],
+	['DeInferno',    {{.UserStatsForGame.Stats.TotalWinsMapDeInferno}},    {{.UserStatsForGame.Stats.TotalRoundsMapDeInferno}}    - {{.UserStatsForGame.Stats.TotalWinsMapDeInferno}}],
+	['DeLake',       {{.UserStatsForGame.Stats.TotalWinsMapDeLake}},       {{.UserStatsForGame.Stats.TotalRoundsMapDeLake}}       - {{.UserStatsForGame.Stats.TotalWinsMapDeLake}}],
+	['DeNuke',       {{.UserStatsForGame.Stats.TotalWinsMapDeNuke}},       {{.UserStatsForGame.Stats.TotalRoundsMapDeNuke}}       - {{.UserStatsForGame.Stats.TotalWinsMapDeNuke}}],
+	['DeSafehouse',  {{.UserStatsForGame.Stats.TotalWinsMapDeSafehouse}},  {{.UserStatsForGame.Stats.TotalRoundsMapDeSafehouse}}  - {{.UserStatsForGame.Stats.TotalWinsMapDeSafehouse}}],
+	['DeShorttrain', {{.UserStatsForGame.Stats.TotalWinsMapDeShorttrain}}, {{.UserStatsForGame.Stats.TotalRoundsMapDeShorttrain}} - {{.UserStatsForGame.Stats.TotalWinsMapDeShorttrain}}],
+	['DeStmarc',     {{.UserStatsForGame.Stats.TotalWinsMapDeStmarc}},     {{.UserStatsForGame.Stats.TotalRoundsMapDeStmarc}}     - {{.UserStatsForGame.Stats.TotalWinsMapDeStmarc}}],
+	['DeSugarcane',  {{.UserStatsForGame.Stats.TotalWinsMapDeSugarcane}},  {{.UserStatsForGame.Stats.TotalRoundsMapDeSugarcane}}  - {{.UserStatsForGame.Stats.TotalWinsMapDeSugarcane}}],
+	['DeTrain',      {{.UserStatsForGame.Stats.TotalWinsMapDeTrain}},      {{.UserStatsForGame.Stats.TotalRoundsMapDeTrain}}      - {{.UserStatsForGame.Stats.TotalWinsMapDeTrain}}],
+	['DeVertigo',    {{.UserStatsForGame.Stats.TotalWinsMapDeVertigo}},    {{.UserStatsForGame.Stats.TotalRoundsMapDeVertigo}}    - {{.UserStatsForGame.Stats.TotalWinsMapDeVertigo}}]
+]
 
-		var data = google.visualization.arrayToDataTable([
-		  ['Map', 'Wins'],
+dataMaps.sort((a,b) => (b[1] + b[2]) - (a[1] + a[2]) )
+var chartMaps = document.getElementById("container-maps");
+var myChartMaps = echarts.init(chartMaps);
+var app = {};
+optionChartMaps = null;
+optionChartMaps = {
+	title: {
+		text: 'Rounds played per map'
+	},
```

Comment:
```suggestion
```
I think we should remove the title here, since this is the "maps information view" anyway and it's clear what the graph means. Let me know if you think otherwise.